config/samples/osp-director_v1beta1_openstackcontrolplane.yaml

```diff
@@ -7,13 +7,13 @@ spec:
   openStackClientImageURL: quay.io/openstack-k8s-operators/tripleo-deploy:latest
   controller:
     Role: Controller
-    controllerCount: 1
+    controllerCount: 0
     cores: 2
     memory: 20
     diskSize: 50
     baseImageURL: http://download.eng.brq.redhat.com/brewroot/packages/rhel-guest-image/8.3/417/images/rhel-guest-image-8.3-417.x86_64.qcow2
     storageClass: host-nfs-storageclass
-    #baseImageVolumeName: #optional
+    baseImageVolumeName: openstack-base-img #optional
```

Comment:
This will change the default OperatorHub example for this CR. Specifically this gets injected via alm-examples into the resulting CSV when we built the operator... so I'm not sure we'd want both baseImageURL and baseImageVolumeName specified as if that is the case only baseImageVolumeName gets applied.

Reply:
Don't worry, I have that covered by `kustomize` patches that are applied when the CSV is generated